alpha/external-web/server.js

```diff
@@ -9,6 +9,7 @@ var bodyParser = require('body-parser')
 var browserSync = require('browser-sync')
 var config = require('./app/config.js')
 var utils = require('./lib/utils.js')
+var expressValidator = require('express-validator')
```

Comment:
as discussed, might be better to use base validator library rather than include as middleware